src/mixins/resizeConfig.js

```diff
@@ -464,12 +459,20 @@ export default {
     },
     addResizeAnchors() {
       this.anchorPoints.forEach(button => {
-        button.attr('root/display', 'initial');
+        try {
+          this.shape.embed(button);
+        } catch (error) {
+          /* There is an error when re-adding points after removing/re-adding the pool.
+           * Ignore it for now. */
+        }
+
+        button.addTo(this.graph);
+        button.toFront();
```

Comment:
The left corner handles were hidden behind the pool, so I had to call this to ensure it was always visible.